test/Microsoft.ML.TestFramework/Attributes/OnnxFactAttribute.cs

```diff
@@ -12,14 +12,15 @@ namespace Microsoft.ML.TestFramework.Attributes
     /// </summary>
     public sealed class OnnxFactAttribute : EnvironmentSpecificFactAttribute
     {
-        public OnnxFactAttribute() : base("Onnx is 64-bit Windows only")
+        public OnnxFactAttribute() : base("Onnx is not supported on Linux with libc < v2.23")
         {
         }
 
         /// <inheritdoc />
         protected override bool IsEnvironmentSupported()
         {
```

Comment:
If we are going ahead with this change right now, can you please make the corresponding change in Microsoft.ML.Tests\OnnxConversionTest.cs in the function IsOnnxRuntimeSupported?

This release doesn't add any extra capabilities that ML.NET needs (other than x86 support - Do we have any numbers for x86 consumers of ML.NET? ). But a major release of ORT is forthcoming very soon and that has a lot more fixes and features we need.  I would prefer to wait for that release to make the next major upgrade and announce that along with all the new transformers that would support exporting to Onnx.

But if you think this is important to get through right now, please make the change above and I will sign off on the PR.

Reply:
> I would prefer to wait for that release to make the next major upgrade

Why would we wait? If there is a new version now - why not get the test coverage now? Especially since the work to upgrade now is very minimal - changing a version number.

> Do we have any numbers for x86 consumers of ML.NET? 

This came up in the context of using ML.NET in Azure Functions. Someone pointed out me to https://github.com/dotnet/machinelearning#operating-systems-and-processor-architectures-supported-by-mlnet

> 32 bit is supported on Windows, except for TensorFlow, LightGBM, and ONNX related functionality.

ONNX functionality supports x86 (even in the current `0.4.0` version). This reminds me, I should update that line in the README in this PR as well.

Reply:
> can you please make the corresponding change in Microsoft.ML.Tests\OnnxConversionTest.cs in the function IsOnnxRuntimeSupported?

Done. I consolidated the logic into a single place.

Reply:
> My impression is that deployments of 32bit processors and 32 bit Windows is steadily shrinking.

While that is true, there are still applications/processes that are 32-bit. Take, for example, Visual Studio. Even though your OS and machine are 64-bit, the Visual Studio process itself is 32-bit. (Visual Studio is just an example here - there are plenty of other applications that are 32-bit. See #97.)

> Do you know where we can get the numbers from?

I don't.